### ISSUE TITLE
MGLMapView.annotation(at point:NSPoint) not working

### ISSUE THREAD

**Issue by kwinterG** [mapbox/mapbox-gl-native-ios#305]
I am trying to use the MGLMapView.annotation(at point:NSPoint) method to find MGLPointAnnotations that I have on the map.    But I cannot get that to work.   The sample code and print output are below:

       let lat = clickCoordinate!.latitude
        let lng = clickCoordinate!.longitude
        print("Pin to drop at \(lng), \(lat)")
        
        pointInFocus = MGLPointAnnotation()
        pointInFocus!.coordinate = clickCoordinate!
        pointInFocus!.title = "Test point"
        mapView.addAnnotation(pointInFocus!)        
        // just to check if we can find it
        print("Created annotation : \(pointInFocus)")
        let pointLoc = mapView.convert(clickCoordinate!, toPointTo: mainWindow.contentView)
        let ann = mapView.annotation(at: pointLoc)
        if ann != nil {
            print("Coorindate : \(clickCoordinate!) is point: \(pointLoc) has annotation > \(ann)")
        }
        else {
            print("Coorindate : \(clickCoordinate!) is point: \(pointLoc) has no annotation")
        }

Print output: 

Pin to drop at -79.61583380719357, 43.67737930724692
Created annotation : Optional(<MGLPointAnnotation: 0x600000cc8570; title = "Test point"; subtitle = (null); coordinate = 43.677379, -79.615834>)
Coorindate : CLLocationCoordinate2D(latitude: 43.67737930724692, longitude: -79.61583380719357) is point: (691.000000000006, 413.0) has no annotation

### Steps to reproduce

 1.  See code fragment supplied

### Expected behavior

The method should be able to find the newly created MGLPointAnnotation and return by the call

### Actual behavior

Method returns nil

### Configuration

**Mapbox SDK versions: 0.16.0**
**iOS/macOS versions: OSX 10.14.6** 
**Device/simulator models:**
**Xcode version: 10.3**

**Comment by 1ec5** [comment]
This is probably related to mapbox/mapbox-gl-native-ios#306 (which I just moved over from the gl-native repository). Does it happen for you consistently? Do you see any other performance issues along the lines of mapbox/mapbox-gl-native#7820?

**Comment by kwinterG** [comment]
Yes, I can reproduce this consistently.  Similar issue also happens if I try to use: MGLMapVIew.visibileAnnotationsInRect(rec: CGRect) will return nil.

I haven't done a test with massive number of annotations.  For the tests I did,  I did not see performance issues along lines of mapbox/mapbox-gl-native#7820

But I do see the same problem as https://github.com/mapbox/mapbox-gl-native/issues/11275 which was referred  to by #306
